kyuubi-server/src/main/scala/org/apache/kyuubi/engine/spark/SparkProcessBuilder.scala

```diff
@@ -168,6 +168,13 @@ object SparkProcessBuilder {
   final val APP_KEY = "spark.app.name"
   final val TAG_KEY = "spark.yarn.tags"
   final val MASTER_KEY = "spark.master"
+  final val PATH_CONFIGS = Seq(
+    SPARK_FILES,
+    "spark.jars",
+    "spark.archives",
+    "spark.yarn.dist.files",
+    "spark.yarn.dist.jars",
+    "spark.yarn.dist.archives")
```

Comment:
cc @zwangsheng for k8s code path

Reply:
maybe there is no such parameter for k8s？

these config items are from SparkSubmit.

Reply:
Well, I'm not sure if the parameters (etc. `spark.kubernetes.kerberos.krb5.path`) for uploading functional files are in your range.

Reply:
seems that spark.kubernetes.kerberos.krb5.path is not uploaded during spark-submit?

It should be including in the image?

Reply:
![image](https://user-images.githubusercontent.com/6757692/177071108-bdff5de4-e77f-439f-ad4d-caf3d57dd728.png)

The config item I use is from SparkSubmit

Reply:
This conf `spark.kubernetes.kerberos.krb5.path` used in `KerberosConfDriverFeatureStep` which build driver pod.
This file is uploaded when the Driver Pod Resource is built after Submit and becomes a mount to DriverPod

Reply:
I see, I did not enable `kubenertes` profile in my spark project, let me have a check.

Reply:
BTW, we can add some comments of where the spark code reference it to help recall in future

Reply:
```
spark.kerberos.keytab
spark.yarn.keytab
spark.kubernetes.kerberos.krb5.path
```

Reply:
ok, have added above config item and add some comments

Reply:
spark.kubernetes.file.upload.path

Reply:
What is our attitude on this `spark.kubernetes.file.upload.path `, in Kubernetes Cluster mode, that this configuration is important for uploading jars.

Reply:
We also need to check  kubernetes authenticate etc. `spark.kubernetes.authenticate.driver.caCertFile`.

Reply:
Ok, we can also add it into path configs.

Reply:
Thanks @zwangsheng , is there any other parameter for k8s use case?

Reply:
This is the relevant configuration I can find, based on the situations I have used.

Reply:
ok, we can complete it if we meet another case in the future.